bundles/com.espressif.idf.debug.gdbjtag.openocd/src/com/espressif/idf/debug/gdbjtag/openocd/utils/EspConfigParser.java

```diff
@@ -0,0 +1,65 @@
+package com.espressif.idf.debug.gdbjtag.openocd.utils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.JSONValue;
+
+import com.espressif.idf.core.logging.Logger;
+
+public class EspConfigParser
+{
+	@SuppressWarnings("unchecked")
+	public ArrayList<String> getTargets()
```

Comment:
Use generic List<String> instead of ArrayList<String>

Reply:
changed it, thx